ci/docker/install/install_openblas.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
Please rename file to match the style of our filenames. "install_platform_action"

Reply:
Platform is universal, and [none of the file names have a install prefix](https://github.com/apache/incubator-mxnet/tree/master/ci/docker/install). That gives "action" as the only needed suffix. What exactly the name you are suggesting in this case?

Reply:
Sorry, it was supposed to be "install/platform_action". 

I'd propose arm_openblas.sh

Reply:
It does not contain anything related to arm, even if we use it only for such cases.

Reply:
Well I would have proposed something related to cross-compilation, but that might be confusing. You're right that it's not arm specific, but neither is it general :/

Reply:
I would prefer to keep it that way, would that be alright?

Reply:
I would prefer a rename since this script is not general purpose. It's only applicable for crosscompilation. Your proposed naming would indicate that it's usable for everything.

Additionally, we only use it for crosscompilation on ARM as of now, so we should mark it accordingly. If we expand the scope later on, we're free to rename it again. For now, it's limited to the crosscompilation scope of ARM and should thus be properly named.

Reply:
What do you see in this script as not general purpose?

Reply:
```PREFIX=${CROSS_ROOT} make install```

Reply:
Ok, I see thanks